src/main/java/frc/robot/subsystems/led/LED.java

```diff
@@ -16,17 +15,17 @@ public class LED extends SubsystemBase {
   private final AddressableLED m_LEDStrip;
   private final AddressableLEDBuffer m_LEDStripBuffer;
   private final Color[] m_colors;
-  private boolean currentToggle = true;
-  private boolean m_currentCube = false;
-  private boolean m_currentCone = false;
+  // private boolean currentToggle = true;
+  private boolean m_toggleCube = false;
+  private boolean m_toggleCone = false;
```

Comment:
These fields aren't generally something we want to store in the subsystem, but rather a command. The subsystem should simply be concerned with the color that it is displaying, and that is it. We can use commands to tell the LED subsystem to display a certain color. If you want to achieve a toggle effect, you can use triggers and commands to toggle the colors